src/main/java/randoop/reflection/ClassLiteralExtractor.java

```diff
@@ -18,8 +20,16 @@
 
   private MultiMap<ClassOrInterfaceType, Sequence> literalMap;
 
-  ClassLiteralExtractor(MultiMap<ClassOrInterfaceType, Sequence> literalMap) {
+  /**
+   * The term frequency mapping is used for the weighted constants option. This will keep track of
+   * how often a constant appears across all constants.
```

Comment:
"across all constants": should this be "across all classes", or some other term?  If the current wording is correct, please clarify.